lowstate_odom:
  ros__parameters:
    dds_domain_id: 0
    dds_topic: "rt/lowstate"
    ros_topic: "/dog_odom"
    odom_frame: "odom"
    base_frame: "base_link"

livox_bridge:
  ros__parameters:
    dds_domain_id: 0
    dds_topic: "rt/utlidar/cloud_livox_mid360"
    ros_topic: "/livox/lidar"
    frame_id: "livox_mid360"
